source/Weapon.h

```diff
@@ -192,6 +192,12 @@ class Weapon {
 	// Calculate the percent damage that this weapon deals given the distance
 	// that the projectile traveled if it has a damage dropoff range.
 	double DamageDropoff(double distance) const;
+	// Return the weapon's damage dropoff at maximum range
+	double MaxDropoff() const;
+	// Return the range at which the weapon's damage dropoff begins
+	double DropoffStart() const;
+	// Return the range at which the weapon's damage dropoff ends
+	double DropoffEnd() const;
```

Comment:
No need to have separate functions for the start and the end when the variable is stored as a range/pair internally anyways.